Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1401,6 +1401,17 @@ extension BrowserViewController: WKNavigationDelegate {
                     return
                 }
 
+                if let readerMode = tab.getHelper(name: ReaderMode.name()) as? ReaderMode {
```

Comment:
Why is this inside of the `dispatch_after` block? That block is here only as a workaround for the taking a screenshot.